### DISCUSSION TITLE
Genre Ideas

### DISCUSSION THREAD

**Discussion by dimensionscape** [haxecollab/ludum-dare-56#7] [Ideas]
Game genres vary widely often sharing or mixing traits to create new genres all together. While we can't design a game until the jam starts, we can save some time by narrowing down the scope of the game by selecting some genre constraints.

I would like everyone to select up to three genre's as your preference(in order):

```
1. XXX 
2. XXX
3. XXX
```

**You are not restricted to those in this list below, please list whatever you come up with, even if it is a brief description of a new and exciting genre you've thought up.**

# Game Genres

## 1. Action
- **Platformer** (e.g., *Super Mario Bros.*, *Celeste*)
- **Shooter**
  - First-person shooter (FPS) (e.g., *Call of Duty*, *Halo*)
  - Third-person shooter (e.g., *Gears of War*, *Fortnite*)
- **Fighting** (e.g., *Street Fighter*, *Tekken*)
- **Beat ‘em up** (e.g., *Streets of Rage*, *Castle Crashers*)
- **Stealth** (e.g., *Metal Gear Solid*, *Hitman*)
- **Survival** (e.g., *The Forest*, *Don’t Starve*)
- **Rhythm** (e.g., *Guitar Hero*, *Dance Dance Revolution*)

## 2. Adventure
- **Action-adventure** (e.g., *The Legend of Zelda*, *Tomb Raider*)
- **Survival horror** (e.g., *Resident Evil*, *Silent Hill*)
- **Metroidvania** (e.g., *Hollow Knight*, *Metroid*)
- **Text adventure/Interactive fiction** (e.g., *Zork*, *80 Days*)
- **Graphic adventure** (e.g., *Monkey Island*, *The Walking Dead*)

## 3. Role-Playing Games (RPG)
- **Action RPG** (e.g., *The Witcher*, *Dark Souls*)
- **Turn-based RPG** (e.g., *Final Fantasy*, *Persona*)
- **Tactical RPG** (e.g., *Fire Emblem*, *Final Fantasy Tactics*)
- **MMORPG** (e.g., *World of Warcraft*, *Final Fantasy XIV*)
- **Roguelike** (e.g., *Rogue*, *Hades*)
- **Dungeon Crawler** (e.g., *Diablo*, *Torchlight*)

## 4. Simulation
- **Life simulation** (e.g., *The Sims*, *Animal Crossing*)
- **Management simulation** (e.g., *Cities: Skylines*, *RollerCoaster Tycoon*)
- **Vehicle simulation** (e.g., *Microsoft Flight Simulator*, *Gran Turismo*)
- **Sports simulation** (e.g., *FIFA*, *Madden NFL*)
- **Farming simulation** (e.g., *Stardew Valley*, *Harvest Moon*)

## 5. Strategy
- **Real-time strategy (RTS)** (e.g., *StarCraft*, *Age of Empires*)
- **Turn-based strategy (TBS)** (e.g., *Civilization*, *XCOM*)
- **4X (eXplore, eXpand, eXploit, eXterminate)** (e.g., *Civilization*, *Stellaris*)
- **Tower defense** (e.g., *Plants vs. Zombies*, *Kingdom Rush*)
- **Auto battler** (e.g., *Teamfight Tactics*, *Auto Chess*)

## 6. Sports
- **Traditional sports** (e.g., *FIFA*, *NBA 2K*)
- **Extreme sports** (e.g., *Tony Hawk’s Pro Skater*, *SSX*)
- **Racing** (e.g., *Mario Kart*, *Forza Horizon*)
- **Combat sports** (e.g., *WWE 2K*, *UFC*)

## 7. Puzzle
- **Logic puzzle** (e.g., *Tetris*, *Sudoku*)
- **Physics puzzle** (e.g., *Portal*, *Angry Birds*)
- **Match-3** (e.g., *Candy Crush*, *Bejeweled*)
- **Escape room** (e.g., *The Room*, *Myst*)

## 8. Sandbox/Open World
- **Sandbox** (e.g., *Minecraft*, *Terraria*)
- **Open-world** (e.g., *The Elder Scrolls V: Skyrim*, *Grand Theft Auto V*)

## 9. Party/Board/Card
- **Party** (e.g., *Mario Party*, *Jackbox Party Pack*)
- **Board game adaptations** (e.g., *Catan*, *Monopoly Plus*)
- **Card games** (e.g., *Hearthstone*, *Slay the Spire*)

## 10. Idle/Incremental
- **Clicker** (e.g., *Cookie Clicker*, *Adventure Capitalist*)
- **Idle RPG** (e.g., *AFK Arena*, *Idle Heroes*)

## 11. Educational
- **Edutainment** (e.g., *The Oregon Trail*, *Brain Age*)
- **Training** (e.g., *Duolingo*, *Khan Academy apps*)

## 12. Multiplayer Genres
- **Battle Royale** (e.g., *Fortnite*, *PUBG*)
- **MOBA (Multiplayer Online Battle Arena)** (e.g., *League of Legends*, *Dota 2*)
- **Co-op** (e.g., *Overcooked*, *Left 4 Dead*)
- **MMO (Massively Multiplayer Online)** (e.g., *World of Warcraft*, *Guild Wars 2*)

## 13. Virtual Reality (VR)
- **VR Action** (e.g., *Beat Saber*, *Superhot VR*)
- **VR Adventure** (e.g., *Half-Life: Alyx*, *Lone Echo*)

---

Again, this list isn't exhaustive, and many games blend elements from multiple genres, creating hybrids or entirely new genres.

**Comment by EyeDaleHim** [comment]
1. Stealth
2. Sports
3. Real-time strategy
